Homework-№1-from-07.09.2022/task8.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+
+void task8(int *counterZero, int *counterOfNumbers, int *nowElement)

```

Comment:
```suggestion
int zerosCounter(int *counterZero, int *counterOfNumbers, int *nowElement)
```
Более содержательное имя должно быть.

Reply:
И зачем первый считанный элемент передавать сюда, если его можно считать внутри функции?